src/Hug.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Card from './Card'
+import Flex from './Flex'
+import Icon from './Icon'
+import Text from './Text'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+const StyledCard = styled(Card)`
+  border-radius: ${props => props.theme.radius};
+`
+
+const Hug = props => (
+  <StyledCard {...props} borderColor={props.bg}>
+    <Flex p={3} align="center">
+      {!!props.iconName && <Icon mr={3} name={props.iconName} size={17} />}
```

Comment:
Size 17 will probably get some weird sub pixel rendering/blurriness since most icons are built on 16x16, 24x24, or 32x32 grids

Reply:
I dig it :shovel: